src/routing/approved.rs

```diff
@@ -824,7 +824,8 @@ impl Approved {
             .section
             .chain()
             .keys()
-            .chain(self.network.keys().map(|(_, key)| key));
+            .chain(self.network.keys().map(|(_, key)| key))
```

Comment:
`self.network.keys()` may already contains the `genesis_key` ?

Reply:
Yes, also `section.chain()` as well but there is no harm in that.

Reply:
the type is `vec` ? or it will be `filtered out` later on when map to `BTreeSet` ?

Reply:
It's converted to `HashSet` in `SectionChain::check_trust`.